pkg/clusteragent/autoscaling/workload/external/telemetry.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+package external
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/telemetry"
+	le "github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver/leaderelection/metrics"
+)
+
+const (
+	subsystem = "autoscaling_workload_external"
+)
+
+var (
+	commonOpts = telemetry.Options{NoDoubleUnderscoreSep: true}
+
+	// telemetryHorizontalExternalRecommendations tracks external horizontal scaling recommendation values
+	telemetryHorizontalExternalRecommendations = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"horizontal_scaling_recommended_replicas",
```

Comment:
nit: `horizontal` and `replicas` is redundant, can just be `recommended_replicas`. Providing the recommender host as tag could be valuable as well.